### PR TITLE
Added Integration Test

### PR DESCRIPTION
**Pull request by athakor**

### CODE REVIEW

File: google-cloud-dataproc/src/test/java/com/google/cloud/dataproc/v1beta2/it/ITSystemTest.java [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.dataproc.v1beta2.it;
+
+import static org.junit.Assert.assertEquals;
```
Comment (rahulKQL) [inline google-cloud-dataproc/src/test/java/com/google/cloud/dataproc/v1beta2/it/ITSystemTest.java:18]: If Truth available, Please use that.

File: pom.xml [hunk 2]
```diff
@@ -141,6 +141,11 @@
         <artifactId>proto-google-common-protos</artifactId>
         <version>${google.common-protos.version}</version>
       </dependency>
+      <dependency>
+        <groupId>com.google.cloud</groupId>
+        <artifactId>google-cloud-core</artifactId>
+        <version>${google.core.version}</version>
+      </dependency>
```
Comment (rahulKQL) [inline pom.xml:148]: Did you mean?
```xml
      <dependency>
        <groupId>com.google.cloud</groupId>
        <artifactId>google-cloud-core</artifactId>
        <version>${google.core.version}</version>
        <scope>test</scope>
      </dependency>
```

File: google-cloud-dataproc/src/test/java/com/google/cloud/dataproc/v1beta2/it/ITSystemTest.java [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+    clusterClient.createClusterAsync(CREATE_CLUSTER_REQUEST).get();
+
+    /** create workflow */
+    workflowClient = WorkflowTemplateServiceClient.create();
+    workflowClient.createWorkflowTemplate(PARENT, WORKFLOW_TEMPLATE);
+  }
+
+  @AfterClass
+  public static void afterClass() {
+
```
Comment (rahulKQL) [inline google-cloud-dataproc/src/test/java/com/google/cloud/dataproc/v1beta2/it/ITSystemTest.java]: remove this whitespace

**Review by rahulKQL — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 2 above)
Comment (athakor) [reply pom.xml:148]: > Did you mean?
> 
> ```
>       <dependency>
>         <groupId>com.google.cloud</groupId>
>         <artifactId>google-cloud-core</artifactId>
>         <version>${google.core.version}</version>
>         <scope>test</scope>
>       </dependency>
> ```

if you want to used `ServiceOptions.getDefaultProjectId()` in this repo then we have to expliccitly add it,same i did it for othe client as well.

File: google-cloud-dataproc/src/test/java/com/google/cloud/dataproc/v1beta2/it/ITSystemTest.java (hunk 1 above)
Comment (athakor) [reply google-cloud-dataproc/src/test/java/com/google/cloud/dataproc/v1beta2/it/ITSystemTest.java:18]: no there is no Truth available

File: pom.xml (hunk 2 above)
Comment (rahulKQL) [reply pom.xml:148]: But scope can be test right? Is it necessary to bundle with client?

Comment (rahulKQL) [reply pom.xml:148]: just try with adding `<scope>test</scope>`, If works then you are all set.

File: google-cloud-dataproc/src/test/java/com/google/cloud/dataproc/v1beta2/it/ITSystemTest.java (hunk 3 above)
Comment (athakor) [reply google-cloud-dataproc/src/test/java/com/google/cloud/dataproc/v1beta2/it/ITSystemTest.java]: done

File: pom.xml (hunk 2 above)
Comment (athakor) [reply pom.xml:148]: jeff suggest me in datacatalog,see here same thing i did for it.
https://github.com/athakor/java-datacatalog/blob/master/pom.xml#L141

https://github.com/googleapis/java-datacatalog/blob/master/google-cloud-datacatalog/pom.xml#L77

**Review by pmakani — APPROVED** [review]
LGTM

File: pom.xml (hunk 2 above)
Comment (rahulKQL) [reply pom.xml:148]: Ok, I understand your point 👍

**Review by rahulKQL — APPROVED** [review]

**Pull request closed without merging**